Sources/Sentry/Public/SentrySampleDecision.h

```diff
@@ -0,0 +1,22 @@
+#import <Foundation/Foundation.h>
+/**
+ * Trace sample decision flag.
+ */
+typedef NS_ENUM(NSUInteger, SentrySampleDecision) {
+    /**
+     * Used when the decision to sample a trace should be postpone.
+     */
+    kSentrySampleDecisionUndecided,
+
+    /**
+     * The trace should be sampled.
+     */
+    kSentrySampleDecisionYes,
+
+    /**
+     * The trace should not be sampled.
+     */
+    kSentrySampleDecisionNo
```

Comment:
l: as the class is already called `SentrySampleDecision`, do we need to prefix everything with `kSentrySampleDecision`?

Reply:
thanks